### ISSUE TITLE
Add `Subscribe` button to blog

### ISSUE THREAD

**Issue by MaxGhenis** [PolicyEngine/policyengine-app#99]
I think it would be clearer if the subscribe text were instead a `Subscribe` button below the form.
<img width="302" alt="image" src="https://user-images.githubusercontent.com/6076111/209566284-c3576ab2-fcfe-4165-a3f6-2d22dfc3fa0c.png">

**Comment by nehashinde9918** [comment]
I can start with this issue.

**Comment by MaxGhenis** [comment]
Thank you @nehashinde9918! @ibushong recently shared this feedback, could you please integrate it?
>Some (minor) feedback on the Blog section of the home page on mobile:
I think this should be refactored so that the “Blog” header text is above the slider/carousel. And then the Subscribe CTA/field could be below the carousel. Feels like too much empty space in the current implementation, and the first blog entry is past the (horizontal) fold

![image](https://user-images.githubusercontent.com/6076111/214702177-6f6c6625-bbf1-431a-ac07-10a74f00acb8.png)

**Comment by veektrie** [comment]
has this issue been fixed?

**Comment by MaxGhenis** [comment]
No - @nehashinde9918 do you still intend to work on it?

**Comment by nehashinde9918** [comment]
Yes, I have started working on it. Will finish it soon and update.
